src/utils/Transactions.ts

```diff
@@ -12,7 +12,7 @@ export function loadOrCreateTransaction(eth_transaction : ethereum.Transaction,
     transaction.from = eth_transaction.from
     transaction.to = eth_transaction.to
     transaction.value = eth_transaction.value
-    transaction.gasUsed = eth_transaction.gasUsed
+    // transaction.gasUsed = eth_transaction.gasUsed
```

Comment:
why is this commented out

Reply:
according to the latest version of `@graphprotocol/graph-ts`, should use `gasLimit` instead
https://github.com/graphprotocol/graph-node/issues/2619